tests/sentry/migrations/test_0474_make_organization_mapping_org_id_unique.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from django.db import IntegrityError
+
+from sentry.models import Organization, OrganizationMapping
+from sentry.testutils.cases import TestMigrations
+
+
+class MakeOrganizationMappingOrgIdUnique(TestMigrations):
```

Comment:
Nit: You probably don't need a test for adding a unique, we can rely on django doing this correctly. These are normally used for backfills